pennylane/tape/qscript.py

```diff
@@ -295,6 +295,26 @@ def circuit(self):
         """
         return self.operations + self.measurements
 
+    @property
+    def prep(self) -> List[Operator]:
+        """
+        Returns the state preparations on the quantum script.
+
+        Returns:
+            list[.Operator]: state preparations
+        """
+        return self._prep
+
+    @property
+    def gates(self) -> List[Operator]:
+        """
+        Returns the operations (excluding state preparations) on the quantum script.
+
+        Returns:
+            list[.Operator]: quantum operations
+        """
+        return self._ops
+
```

Comment:
I think I'm good with adding these as public properties, but we should break these changes out into a separate PR so we can focus on them.